angular-chart.js

```diff
@@ -321,12 +325,16 @@
       });
     }
 
-    function setLegend (elem, chart) {
+    function setLegend (elem, chart, scope) {
       var $parent = elem.parent(),
-          $oldLegend = $parent.find('chart-legend'),
-          legend = '<chart-legend>' + chart.generateLegend() + '</chart-legend>';
-      if ($oldLegend.length) $oldLegend.replaceWith(legend);
-      else $parent.append(legend);
+          $oldLegend = $parent.find('chart-legend');
+      if (scope.legend) {
```

Comment:
Actually that's the whole point of the PR. The attribute was interpreted as a string because of the ``@`` attribute.